test/unit/reflectivity_ui/interfaces/data_handling/test_data_manipulation.py

```diff
@@ -68,6 +120,30 @@ def test_smart_stitch_reflectivity(self, data_server, mocker_file_open, stitchin
         scaling_factors = smart_stitch_reflectivity(manager.reduction_list, None, False, 0.008)
         assert scaling_factors == pytest.approx([1.0, 0.1809, 0.1354], abs=0.001)
 
+    @pytest.mark.parametrize(
+        "normalize_to_unity, q_cutoff, global_fit, expected_scaling_factors",
+        [
+            (False, None, False, [1.0, 1.66667, 5.0]),
```

Comment:
I'm confused the test is working. Here `q_cutoff == None` but function `smart_stitch_reflectivity()` does require a numeric value for parameter `q_cutoff`! What am I reading wrong :question: